src/main/java/ma/m3achaba/plantes/controller/AuthController.java

```diff
@@ -0,0 +1,79 @@
+package ma.m3achaba.plantes.controller;
+
+
+
+import ma.m3achaba.plantes.dto.RegisterRequest;
+import ma.m3achaba.plantes.model.User;
+import ma.m3achaba.plantes.services.imp.AuthenticationUserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class AuthController {
```

Comment:
seperate UserController(findAll,updateUser,deleteUser,getProfile,updateProfile,saveUser) and ¡AuthController(register,loginrefresh)